ffig/generators/python/python2.py

```diff
@@ -0,0 +1,5 @@
+from generators import default_generator
+
+def py2_generator(binding, api_classes, env, args, output_dir):
```

Comment:
We're currently building universal python bindings. That won't be the case for long though.